web/src/components/forms/project-upload-form.tsx

```diff
@@ -124,6 +131,13 @@ export const ProjectUploadForm: FC<Props> = ({ onHide }) => {
         placeholder="Describe your PEP."
         {...register('description')}
       ></textarea>
+      <div className="mt-3">
+       <SchemaDropdown
```

Comment:
Using custom components with `react-hook-form` (`useForm`) is pretty difficult and annoying. I still don't know how to do it fully. I use a lot of trial and error. Here is an example of how I got it to work with the validator form: https://github.com/pepkit/pephub/blob/439fd53ffea6199c39fbd091a486aeb7ae09e917/web/src/components/forms/validator-form.tsx#L99

You have to use a special `<Controller />` component from `react-hook-form`:

```tsx
<Controller
              name="pepRegistryPath"
              control={control}
              render={({ field }) => (
                <Select
                  {...field}
                  isClearable
                  placeholder="Select a PEP"
                  className="mt-2"
                  // @ts-ignore
                  options={
                    projects?.items.map((project) => ({
                      value: `${project.namespace}/${project.name}:${project.tag}`,
                      label: `${project.namespace}/${project.name}:${project.tag}`,
                    })) || []
                  }
                />
              )}
            />
            ```

Reply:
Ah, I see. Thank you, yeah we can wipe this custom component in favor of the controller one.